members/state_pa_gov_tom_wolf.yaml

```diff
@@ -1,14 +1,17 @@
 bioguide: state_pa_gov_tom_wolf
 contact_form:
   method: post
+  usechrome: true
   steps:
     - visit: "https://www.governor.pa.gov/contact/#OnlineForm"
     - find:
       - selector: "#newsletterModal2 button"
     - click_on:
       - selector: "#newsletterModal2 button"
     - find:
-      - selector: "#newsletterModal2 button"
+      - selector: "#nf-form-421-cont"
+    - find:
+      - selector: "#nf-field-3191"
```

Comment:
do we need both of these?

Reply:
I took the first `find` out and left the one related to the `firstname` selector